Formula/kube-aws.rb

```diff
@@ -0,0 +1,44 @@
+require "yaml"
+
+class KubeAws < Formula
+  desc "Kubernetes on AWS"
+  homepage "https://github.com/coreos/coreos-kubernetes/tree/master/multi-node/aws"
+  url "https://github.com/coreos/coreos-kubernetes/archive/v0.8.1.tar.gz"
+  sha256 "fe65c3fb72af7aa47dad07b8428eace077314edd3c129dba1fea5d3cccfaf094"
+  head "https://github.com/coreos/coreos-kubernetes.git"
+
+  depends_on "go" => :build
+
+  def install
+    executable = "bin/kube-aws"
+    ENV["GOPATH_VENDOR"] = "#{Dir.pwd}/_gopath-vendor"
+    ENV["GOPATH_KUBE_AWS"] = "#{Dir.pwd}/_gopath-kube-aws"
+    ENV["KUBE_AWS_DIR"] = "#{ENV["GOPATH_KUBE_AWS"]}/src/github.com/coreos/coreos-kubernetes/multi-node/aws"
+    ENV["GOPATH"] = "#{ENV["GOPATH_VENDOR"]}:#{ENV["GOPATH_KUBE_AWS"]}"
+
+    Dir.chdir "#{Dir.pwd}/multi-node/aws/"
+
+    rm_rf [ENV["GOPATH_VENDOR"], ENV["GOPATH_KUBE_AWS"]]
```

Comment:
Why are we deleting these then recreating them below?  What do they contain originally?